shell.nix

```diff
@@ -0,0 +1,16 @@
+let
+  docs =
+    import ./default.nix;
+
+  pkgs =
+    docs.pkgs;
+in
+pkgs.mkShell {
+  name = "postgrest-docs";
+
+  buildInputs = [
+    docs.build
+    docs.serve
+    docs.spellcheck
```

Comment:
Do you have any idea how to avoid repetition here? I feel like adding `docs.anotherTool` every time could be avoided?

In javascript terms, I would just like to do `[ ...docs ]`, or something like that. Although one of them is a set, the other one a list..... something like `buildInputs = valuesOf docs` maybe?

Reply:
Not sure what happens when I add `docs.pkgs` to buildInputs as well, though.

Reply:
> Not sure what happens when I add docs.pkgs to buildInputs as well, though.

Could be fun, absolutely all of the 60k+ packages I think :-)

Otherwise you could use `builtins.attrValues` to turn the set from default.nix into a list.

I think it's good to be explicit with what we include in shell.nix. If we have many related scripts, we can group them with pkgs.buildEnv in default.nix.

Reply:
> Could be fun, absolutely all of the 60k+ packages I think :-)

I had a feeling there... :)

Okay, I stick with this then. Thanks!